sdks/python/apache_beam/io/jdbc.py

```diff
@@ -96,25 +98,37 @@
 
 
 def default_io_expansion_service():
-  return BeamJarExpansionService('sdks:java:io:expansion-service:shadowJar')
+    return BeamJarExpansionService(
+        ':sdks:java:extensions:schemaio-expansion-service:shadowJar')
 
 
-WriteToJdbcSchema = typing.NamedTuple(
-    'WriteToJdbcSchema',
+ReadFromWriteToJdbcSchema = typing.NamedTuple(
```

Comment:
nit: WDYT about calling this JdbcConfigSchema?

Reply:
I think the next improvement to SchemaIO/Provider should be to formally differentiate read-only and write-only configuration, it's confusing to combine them this way when it's not needed. The only place its _needed_ is in SQL, so we should do it there.
(you don't need to do anything here, I'm just getting some thoughts out)

Reply:
Filed BEAM-10709 for this

Reply:
I agree it is a more concise way of calling it. I updated the name here.